.github/stale.yml

```diff
@@ -0,0 +1,24 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - bug - failure to compile
+  - bug - ice
+  - bug - incorrect behaviour
+  - bug - memory leak
+  - bug - miscompile
+  - bug - regression
+  - bug - segfault
+  - feature_request
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
Agree with comments in the PR proper, think a `stale` label might be more appropriate? It's not that we `won't fix`, more that just something is probably lacking input/information to permit a classification of the problem?

Reply:
right, so let us make a new label `stale` and then use that. I am not sure this label will see much use anyway?

Reply:
Done! https://github.com/numba/numba/labels/stale

Reply:
Oh, we have a `stale` label?  Does anyone know if it has been there all along or did someone make it recently?

Reply:
It looks like the `stale` label is not used in any issues, so let's use that.

Reply:
Made it for the purpose of this PR!